### DISCUSSION TITLE
后端抓取总是404（已解决，插件配置错误）

### DISCUSSION THREAD

**Discussion by hejunchen** [metatube-community/jellyfin-plugin-metatube#192] [General]
后端服务器抓取数据一直都出 404 错误。没有人更新和修补这些问题吗？

**Comment by xjasonlyu** [comment]
有什么数据404？从来没有遇到此类问题。

**Comment by hejunchen** [comment]
看错误应该是后端服务器那边无法抓取数据。是由上面特殊设置吗？

https://github.com/metatube-community/jellyfin-plugin-metatube/discussions/191
https://github.com/metatube-community/jellyfin-plugin-metatube/discussions/185

**Reply by xjasonlyu** [reply]
我觉得主要还是网络问题，或者一些冷门资源出现了刮削失败的情况，具体不是很清楚，但并没有影响到大多数人的使用。

**Comment by hejunchen** [comment]
部分后端服务器所在的网络由什么要求吗？

https://metatube-community.github.io/wiki/free-servers/
那三个免费的后端服务，我也用过，一开始都能搜刮到资源信息，忽然就不行了，都是404.
我自己本地部分的后端，好像没有成功过一次，都是404返回。
而且其他人也询问同样的问题。

那个 db 文件里面会有记录具体的错误信息吗？比如：network or host unreachable 之类的错误。

**Comment by hejunchen** [comment]
刚才我试着打开那个 db 文件，里面有几十个成功抓到的资源信息。随便挑了一个资源 details 页面 url 打开，是无法显示成功的，总是redirect 我去登录，但是即便是登录了也无法看到资源信息。但是，如果我打开本地的 vpn 切换到日本，再去打开那同一个网页，就能够直接打开。看来的确是网络的问题，后端用的 provider 资源站点似乎非常挑剔 requester 的IP地址啊。

**Comment by hejunchen** [comment]
用日本的IP地址，可以抓到好多个，但过了一阵子，就开始404了。这时对连续抓取资源有数量限制吗？比如：只能连续抓100个？

**Comment by hejunchen** [comment]
@xjasonlyu - 我找到原因了。是我插件里的 provider filter 选项被打勾了，所以过滤绝大多数的资源，而数据库文件里能抓取到的资源都是来自于同一个 provider。

可能是我安装后，无意中把这个选项打勾了，所以绝大多数的资源抓取都是404.
把勾去掉，现在就好多了。基本上都是200返回。而且没有开vpn用日本的地址。

![image](https://github.com/metatube-community/jellyfin-plugin-metatube/assets/6847102/96bd4de2-a131-4ace-a787-805e60dc7e14)

**Comment by southblacks** [comment]
网络问题，我换成美国的vps就好了
